scene/gui/tree.cpp

```diff
@@ -682,7 +697,21 @@ TreeItem *TreeItem::get_prev_visible(bool p_wrap) {
 	return current;
 }
 
-TreeItem *TreeItem::get_next_visible(bool p_wrap) {
+TreeItem *TreeItem::get_prev_visible(bool p_wrap) {
+	TreeItem *loop = this;
+	TreeItem *prev = this->_get_prev_visible(p_wrap);
```

Comment:
It's weird that `_get_prev_visible()` can return a TreeItem that isn't visible. We should rename this method, but it can be done in later PR.

Reply:
Yeah, I agree. I really couldn't think of a better name 😅